src/pages/LoginSignup/LoginSignup.js

```diff
@@ -0,0 +1,28 @@
+import { lazy, Suspense, React } from 'react';
+import { Route, BrowserRouter as Switch } from 'react-router-dom';
+
+import './LoginSignup.scss';
+
+const Login = lazy(() => import('./Login'));
+const Signup = lazy(() => import('./Signup'));
+
+function LoginSignup() {
+  return (
+    <div className="login">
+      <div className="login-bg">
+        <Suspense fallback={<div>Loading</div>}>
```

Comment:
Maybe use a loading spinner instead of a text? [Spinners](https://ant.design/components/spin/#header)